spec/gitlab/request_spec.rb

```diff
@@ -7,7 +7,12 @@
   it { is_expected.to respond_to :post }
   it { is_expected.to respond_to :put }
   it { is_expected.to respond_to :delete }
+
   before do
+    # Prevent tests modifying the `default_params` value from causing cross-test
+    # pollution
+    described_class.default_params.delete(:sudo)
```

Comment:
I was seeing order-dependent spec failures in this file depending on when the test on line 56 ran. This resolves that.